lnrpc/rpc.proto

```diff
@@ -1703,6 +1703,15 @@ message PendingUpdate {
     uint32 output_index = 2;
 }
 
+message ReadyForPsbtFunding {
+    string funding_address = 1;
+    int64 funding_amount = 2;
+}
+
+message ReadyForPsbtSigning {
```

Comment:
Is this used anywhere?

Reply:
Yes, this is returned in the stream answer to the `OpenChannel` call.